src/assets/styles/toolbar.less

```diff
@@ -0,0 +1,39 @@
+﻿#toolbar {
+  height: 50px;
+  right: 65px;
+  position: relative;
+}
+
+#toolbar .drag {
+  -webkit-app-region: drag;
+  height: 100%;
+}
+
+.logo {
+  position: fixed;
+  top: 15px;
+  right: 95px;
+  width: 448px;
+}
+
+.min-button {
+  position: fixed;
+  top: 15px;
+  right: 45px;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
```

Comment:
Actullay, we should have an issue- design adn implement cursors across the whole app